index.html

```diff
@@ -4329,6 +4409,31 @@ <h2 id="examples-tzigla">Tzigla</h2>
 
     <h2 id="changelog">Change Log</h2>
 
+    <b class="header">1.4.0</b> &mdash; <small><i>Apr. 8, 2017</i></small>
+    &mdash; <a href="https://github.com/jashkenas/backbone/compare/1.3.3...1.4.0">Diff</a>
+    &mdash; <a href="https://cdn.rawgit.com/jashkenas/backbone/1.4.0/index.html">Docs</a>
+    <br />
+    <ul style="margin-top: 5px;">
+      <li>
+        Added support for setting instance properties before the constructor in <tt>ES6 classes</tt> with a <tt>preinitialize</tt> method.
+      </li>
+      <li>
+        Added <tt>iterator</tt> support implementing JS Iterable protocol.
+      </li>
+      <li>
+        Fixed a bug with the hash generator.
+      </li>
+      <li>
+        `#listenTo` uses public `#on` method. This keeps interop between Backbone and any other event library (including Node.js).
+      </li>
+      <li>
+        <tt>Collection.get</tt> now checks if obj is a <tt>Model</tt> to allow retrieving models with `attributes` key.
+      </li>
+      <li>
+        We no longer test against IE8 in sauce.
```

Comment:
Add a second sentence explaining what this means, especially for those not familiar with sauce (I'm not convinced we even need to mention sauce here.)